recipes/atomic-counter/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "atomic-counter" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/a/atomic_counter/atomic_counter-{{ version }}.tar.gz
+  sha256: bae04f1b3291eb90faeefa7e82685bc2a93f8dd3604578a60faeda6c9dd21043
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
+    - {{ compiler('c') }}
+    - maturin
+  host:
+    - python
+    - pip
+    - maturin
+  run:
+    - python
+    - python-dateutil
+
+test:
+  imports:
+    - atomic_counter
+  requires:
+    - pip
+  commands:
+    - pip check
+
+outputs:
+  - name: atomic-counter
+  - name: atomic_counter
```

Comment:
Why? It's better to use only one.

Reply:
I fundamentally disagree :smile: this is how pypi does it and i've applied this logic across >20 conda forge projects

Reply:
I'm not saying the recipe is wrong. It's necessary when there are two feedstocks with the two names. However, starting with both seems like an anti-pattern.

Reply:
oh i have a huge backlog of projects i still need to go and add -, _ compatibility to. I consider this the preferred pattern after staging a bunch of these. Some references:

https://github.com/conda-forge/conda-forge.github.io/issues/1797
https://github.com/conda-forge/importlib_metadata-feedstock/pull/107
https://github.com/conda-forge/jupyter-packaging-feedstock/pull/34
https://github.com/conda-forge/staged-recipes/pull/22712
https://github.com/conda-forge/staged-recipes/pull/19624
https://github.com/conda-forge/staged-recipes/pull/19452
https://github.com/conda-forge/staged-recipes/pull/19001
etc etc

Reply:
here is the latest:
https://github.com/conda-forge/arrow_odbc-feedstock/pull/23
https://github.com/conda-forge/duckdb-feedstock/issues/6#issuecomment-1654025349